### PR TITLE
refactor!: removes unsupported apis

### PR DESCRIPTION
**Pull request by YaswanthKumar-eng**
This pr is about removing the not supported client api's for v2.

### CODE REVIEW

**Review by ankitrgadiya — CHANGES_REQUESTED** [review]
Looks good, two suggestions:
* Remove the Documentation pages for the Resources we removed.
* Build CLI against it and run Automation Test Suite.

**Comment by smrutisenapati** [comment]
Once merged I will create a release PR, so that cli PR can be created.
Will post cli automation result soon

File: CHANGELOG/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+### Removed
+
+* remove v1 client and cleanup ([dfd78f0](https://github.com/rapyuta-robotics/rapyuta-io-sdk/commit/dfd78f0ad84a96a72748f20d1f58924b4cccc416))
```
Comment (ankitrgadiya) [inline CHANGELOG/CHANGELOG.md:5]:
```suggestion
* Remove unsupported APIs ([dfd78f0](https://github.com/rapyuta-robotics/rapyuta-io-sdk/commit/dfd78f0ad84a96a72748f20d1f58924b4cccc416))
```

File: rapyuta_io/clients/project.py [hunk 2]
```diff
@@ -18,8 +16,6 @@ class Project(ObjBase):
     """
     Project is an organizational unit and all the resources must belong to a Project.
```
Comment (ankitrgadiya) [inline rapyuta_io/clients/project.py:17]: question: Are we keeping the Project class?

File: sdk_test/run_rio_sdk_test.py [hunk 3]
```diff
@@ -36,16 +36,17 @@ def get_concurrent_test_suites(self):
     def setUpSuite(self):
         self.logger.info('Creating project')
         self.config.create_project()
-        self.logger.info('Creating secrets')
-        self.config.create_secrets()
+        # Secrets not needed for sdk integration tests
+        # self.logger.info('Creating secrets')
+        # self.config.create_secrets()
```
Comment (ankitrgadiya) [inline sdk_test/run_rio_sdk_test.py:41]: todo: Let's remove these lines

File: sdk_test/v2_client.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+from rapyuta_io.utils import prepend_bearer_to_auth_token, RestClient
+from rapyuta_io.utils.rest_client import HttpMethod
+from rapyuta_io.utils.utils import get_api_response_data
+
+
+class V2Client:
```
Comment (ankitrgadiya) [inline sdk_test/v2_client.py:6]: todo: Add the docstring to explain why this was added here to break cyclic dependency between SDK and CLI.

**Comment by ankitrgadiya** [comment]
Let's also wait for the Automation results.

File: rapyuta_io/clients/project.py (hunk 2 above)
Comment (smrutisenapati) [reply rapyuta_io/clients/project.py:17]: Kept the model only as its used in user group. Updated it as per v2 response.

**Review by pallabpain — APPROVED** [review]

**Review by ankitrgadiya — APPROVED** [review]

**Pull request merged**